Developers/Unity Project/Spooksville/Assets/Scripts/AudioManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{
+    public static AudioManager instance;
+
+    public Sound[] sounds;
+
+    private void Start()
+    {
+        if (instance == null)
+        {
+            instance = this;
+            DontDestroyOnLoad(gameObject);
+        }
+    }
+
+    private void Awake()
+    {
+        foreach (Sound sound in sounds)
+        {
+            sound.source = gameObject.AddComponent<AudioSource>();
+            sound.source.clip = sound.clip;
+
+            sound.source.volume = sound.volume;
+            sound.source.pitch = sound.pitch;
+        }
+    }
+
+    public void Play(string name, bool isLooping)
+    {
+        Sound s = Array.Find(sounds, sound => sound.name == name);
+        s.source.Play();
```

Comment:
This works fine but might not scale well to hundreds of sounds. You can also have one audio source and just play the clips on the same source as one shots.